<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sort the numbers</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   
    background-color:rgb(255, 255, 0);   
	font: 900 20pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   
    background-color:rgb(255, 255, 0);   
	font: 900 20pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   
    background-color:rgb(255, 255, 0);   
	font: 900 20pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   
    background-color:rgb(255, 255, 0);   
	font: 900 20pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   
    background-color:rgb(255, 255, 0);   
	font: 900 20pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   
    background-color:rgb(255, 255, 0);   
	font: 900 20pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   
    background-color:rgb(255, 255, 0);   
	font: 900 20pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   
    background-color:rgb(255, 255, 0);   
	font: 900 20pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   
    background-color:rgb(255, 255, 0);   
	font: 900 20pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   
    background-color:rgb(255, 255, 0);   
	font: 900 20pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   
    background-color:rgb(255, 255, 0);   
	font: 900 20pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="text">
     <string>11</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   
    background-color:rgb(255, 255, 0);   
	font: 900 20pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   
    background-color:rgb(255, 255, 0);   
	font: 900 20pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="text">
     <string>14</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   
    background-color:rgb(255, 255, 0);   
	font: 900 20pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="text">
     <string>13</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   
    background-color:rgb(255, 255, 0);   
	font: 900 20pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="text">
     <string>15</string>
    </property>
   </widget>
   <widget class="QLabel" name="timegame">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>322</y>
      <width>37</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>325</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNew_Game"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionNew_Game">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
